src/core/binding.h

```diff
@@ -279,7 +279,9 @@ _IRQL_requires_max_(PASSIVE_LEVEL)
 QUIC_STATUS
 QuicBindingInitialize(
     _In_ const CXPLAT_UDP_CONFIG* UdpConfig,
-    _Out_ QUIC_BINDING** NewBinding
+    _Out_ QUIC_BINDING** NewBinding,
```

Comment:
The out parameter should be last.

Also, not related to your change, but the annotation here should be `_Outptr_`.

Reply:
ACK, next iteration:
- update order of parameters
- update annotation